crates/environ/src/vmoffsets.rs

```diff
@@ -150,6 +159,18 @@ pub trait PtrSize {
         16
     }
 
+    /// Return the size of `VMTagDefinition`.
+    #[inline]
+    fn size_of_vmtag_definition(&self) -> u8 {
+        4
+    }
+
+    /// This is the size of the largest value type (i.e. a V128).
+    #[inline]
+    fn maximum_value_size(&self) -> u8 {
+        self.size_of_vmglobal_definition()
+    }
```

Comment:
This feels a bit suspicious to me so I'm leaving a note to myself here to revisit this once I've seen more of this PR

Reply:
Yes, this is ugly. This is basically just a way of saying "give me the size of a `ValRaw`", without having access to that type here.
The reason we need that size is that all continuation payloads (arguments to continuations, return values from continuations, values provided by `suspend`, etc) are currently written to and then read from buffers containing `ValRaw`. Let's keep this comment open, we will certainly be re-visiting that discussion when you have a look at `instructions.rs`.